calibration/CTDMOG/CGINS-CTDMOG-13560__20170411.csv

```diff
@@ -11,15 +11,15 @@ serial,name,value,notes
 37-13560,CC_j,2.860055e-005,
 37-13560,CC_p_range,1450,
 37-13560,CC_pa0,1.858123e-001,
-37-13560,CC_pa1,4.487070e-003,
+37-13560,CC_pa1,4.487071e-03,
 37-13560,CC_pa2,-3.427646e-011,
 37-13560,CC_ptca0,5.242977e+005,
 37-13560,CC_ptca1,-9.601506e+000,
-37-13560,CC_ptca2,5.152491e-001,
+37-13560,CC_ptca2,5.152492e-01,
 37-13560,CC_ptcb0,2.508413e+001,
 37-13560,CC_ptcb1,2.500000e-005,
 37-13560,CC_ptcb2,0.00E+00,
 37-13560,CC_ptempa0,1.690885e+002,
-37-13560,CC_ptempa1,-6.920906e-002,
+37-13560,CC_ptempa1,-6.920905e-02,
 37-13560,CC_ptempa2,4.025226e-007,
-37-13560,CC_wbotc,3.1059e-008,
+37-13560,CC_wbotc,3.105941e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT